cognite/client/_api/data_modeling/instances.py

```diff
@@ -707,6 +752,7 @@ def search(
             query (str): Query string that will be parsed and used for search.
             instance_type (Literal["node", "edge"]): Whether to search for nodes or edges.
             properties (list[str] | None): Optional array of properties you want to search through. If you do not specify one or more properties, the service will search all text fields within the view.
+            target_units (list[PropertyUnitReference] | None): Properties to convert to another unit. The API can only convert to another unit, if a unit has been defined as part of the type on the underlying container being queried.
```

Comment:
The comma here does not belong (5 other occurrence):
```suggestion
            target_units (list[PropertyUnitReference] | None): Properties to convert to another unit. The API can only convert to another unit if a unit has been defined as part of the type on the underlying container being queried.
```